### ISSUE TITLE
Codable

### ISSUE THREAD

**Issue by avgx** [Astemir-10/EmptySubject#1]
Why members of CategoryMealItem, MealModel are nullable?

It seems that api send not nullable

Why var?

It's better to use immutable structures with members as "let" constants.

**Comment by Astemir-10** [comment]
Yes. I agree the API does not really send null. I fixed everything.
